benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -122,6 +122,45 @@ void GraphicsBenchmarkApp::InitKnobs()
     pFullscreenQuadsSingleRenderpass->SetDisplayName("Single Renderpass");
     pFullscreenQuadsSingleRenderpass->SetFlagDescription("Render all fullscreen quads (see --fullscreen-quads-count) in a single renderpass.");
     pFullscreenQuadsSingleRenderpass->SetIndent(1);
+
+    GetKnobManager().InitKnob(&pRenderOffscreen, "enable-offscreen-render", false);
+    pRenderOffscreen->SetDisplayName("Offscreen render");
+    pRenderOffscreen->SetFlagDescription("Enable render to a offscreen/non-swapchain framebuffer.");
+
+    GetKnobManager().InitKnob(&pBlitOffscreen, "offscreen-blit-to-swapchain", true);
+    pBlitOffscreen->SetDisplayName("Blit to swapchain");
+    pBlitOffscreen->SetFlagDescription("Blit offscreen render result to swapchain.");
+    pBlitOffscreen->SetIndent(1);
+
+    std::array<std::string, std::size(kFramebufferFormatTypes)> formatNames;
+    for (int i = 0; i < std::size(kFramebufferFormatTypes); ++i) {
+        formatNames[i] = ToString(kFramebufferFormatTypes[i]);
+    }
+    formatNames[0] = "Swapchain";
+    GetKnobManager().InitKnob(&pFramebufferFormat, "offscreen-framebuffer-format", 0, formatNames);
+    pFramebufferFormat->SetDisplayName("Framebuffer format");
+    pFramebufferFormat->SetFlagDescription("Select the pixel format used in offscreen framebuffer.");
+    pFramebufferFormat->SetIndent(1);
+
+    std::vector<std::string> resolutionString;
+    mResolutionOptions.push_back(std::pair<int, int>(0, 0)); // Swapchain native
+    resolutionString.push_back("Swapchain");
+    for (const auto& res : kSimpleResolutions) {
+        mResolutionOptions.push_back(res);
+        resolutionString.push_back(std::to_string(res.first) + "x" + std::to_string(res.second));
+    }
+    for (const auto& res : kCommonResolutions) {
+        mResolutionOptions.push_back(res);
+        resolutionString.push_back(std::to_string(res.first) + "x" + std::to_string(res.second));
+    }
+    for (const auto& res : kVRPerEyeResolutions) {
+        mResolutionOptions.push_back(std::pair<int, int>(res.first * 2, res.second));
+        resolutionString.push_back(std::to_string(res.first) + "x2" + "x" + std::to_string(res.second));
+    }
+    GetKnobManager().InitKnob(&pResolution, "offscreen-resolution", 0, resolutionString);
+    pResolution->SetDisplayName("Framebuffer resolution");
```

Comment:
nit: "Offscreen framebuffer resolution"?

Reply:
Changed the command line flag.
The description `Framebuffer resolution` is nested under `Offscreen rendering` on the UI.